apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -229,36 +273,50 @@ fields(layout_builtin_reference) ->
 
 desc(builtin) ->
     ?DESC(builtin);
+desc(builtin_session) ->
+    ?DESC(builtin);
 desc(builtin_local_write_buffer) ->
     ?DESC(builtin_local_write_buffer);
 desc(layout_builtin_wildcard_optimized) ->
     ?DESC(layout_builtin_wildcard_optimized);
+desc(layout_builtin_wildcard_optimized_session) ->
+    ?DESC(layout_builtin_wildcard_optimized);
 desc(_) ->
     undefined.
 
 %%================================================================================
 %% Internal functions
 %%================================================================================
 
-ds_schema(Options) ->
+ds_schema(MainRef, InjectionPoint, Options) ->
     sc(
         hoconsc:union([
-            ref(builtin)
-            | emqx_schema_hooks:injection_point('durable_storage.backends', [])
+            MainRef
+            | emqx_schema_hooks:union_option_injections(InjectionPoint, [])
```

Comment:
This addresses this comment: https://github.com/emqx/emqx-platform/pull/135#discussion_r1514765105